activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/services/audit/AuditProducerIT.java

```diff
@@ -545,6 +551,102 @@ public void shouldEmitEventsForTaskAddDeleteGroupCandidates() {
+
+              List<CloudRuntimeEvent<?, ?>> receivedEvents = streamHandler.getLatestReceivedEvents();
+
+              assertThat(streamHandler.getReceivedHeaders()).containsKeys(ALL_REQUIRED_HEADERS);
+
+              assertThat(receivedEvents)
+              .extracting(event -> event.getEventType().name())
+              .containsExactly(PROCESS_CREATED.name(),
+              				PROCESS_STARTED.name(),
+              				ACTIVITY_STARTED.name()/*start event*/,
+              				BPMNActivityEvent.ActivityEvents.ACTIVITY_COMPLETED.name()/*start event*/,
+              				SEQUENCE_FLOW_TAKEN.name(),
+              				ACTIVITY_STARTED.name());
+
+
+        });
+        
+        startProcessEntity = processInstanceRestTemplate.startProcess(new StartProcessPayloadBuilder()
+																			.withProcessDefinitionKey(SIGNAL_PROCESS)
+															                .withName("processInstanceName2")
+															                .withBusinessKey("businessKey2")
+															                .withVariables(Collections.emptyMap())
+															                .build());
+                
+        //then
+        await().untilAsserted(() -> {
+              assertThat(streamHandler.getReceivedHeaders()).containsKeys(RUNTIME_BUNDLE_INFO_HEADERS);
+
+              List<CloudRuntimeEvent<?, ?>> receivedEvents = streamHandler.getLatestReceivedEvents();
+
+              assertThat(streamHandler.getReceivedHeaders()).containsKeys(ALL_REQUIRED_HEADERS);
+
+              assertThat(receivedEvents)
+              .extracting(event -> event.getEventType().name())
+              .containsExactly(PROCESS_CREATED.name(),
+              				PROCESS_STARTED.name(),
+              				ACTIVITY_STARTED.name()/*start event*/,
+              				BPMNActivityEvent.ActivityEvents.ACTIVITY_COMPLETED.name()/*start event*/,
+              				SEQUENCE_FLOW_TAKEN.name(),
+              				ACTIVITY_STARTED.name());
+
+
+        });
+        
+        
+        SignalPayload signalProcessInstancesCmd = ProcessPayloadBuilder.signal().withName("Test").build();
+
+        //when
+        ResponseEntity<Void> responseEntity = restTemplate.exchange(PROCESS_INSTANCES_RELATIVE_URL + "/signal",
+                                                                    HttpMethod.POST,
+                                                                    new HttpEntity<>(signalProcessInstancesCmd),
+                                                                    new ParameterizedTypeReference<Void>() {
+                                                                    });
+
+        await("Broadcast Signals").untilAsserted(() -> {
+            assertThat(streamHandler.getReceivedHeaders()).containsKeys(RUNTIME_BUNDLE_INFO_HEADERS);
+
+            List<CloudRuntimeEvent<?, ?>> receivedEvents = streamHandler.getLatestReceivedEvents();
+
+            assertThat(streamHandler.getReceivedHeaders()).containsKeys(ALL_REQUIRED_HEADERS);
+
+            assertThat(receivedEvents)
```

Comment:
Can you add another assert to check that `processDefinitionId` and `processInstanceId` are set properly?